src/widgets/_toolbars.scss

```
@@ -15,6 +15,17 @@ actionbar {
     &.flat{
         @extend .inline-toolbar;
     }
+
+    &.title {
+        font-size: 11pt;
+        font-weight: bold;
```

Let's be more specific here to match what we've done in window titles
```suggestion
        font-weight: 600;
```